iml-wasm-components/iml-action-dropdown/src/lib.rs

```diff
@@ -62,6 +62,14 @@ fn show_class(model: &Model) -> &str {
     }
 }
 
+fn composite_ids_to_query_string(compositeIds: Vec<Record>) -> String {
+    compositeIds
+        .iter()
+        .map(|(c, i)| format!("{}:{}", c, i))
```

Comment:
Since you've already created separate modules besides lib, you should organize new code into modules.

Reply:
agreed, where would be a good place for this to live? `composite-transforms.rs`?

Reply:
Also in locks.rs